app/src/App.tsx

```diff
@@ -70,7 +70,7 @@ export const App = (): React.ReactElement => {
         <Route path='/register' page={RegistrationPage} />
         <Route path='/forgot-password' page={ForgotPasswordPage} />
         <Route path='/domains' page={DomainsPage} />
-        <Route path='/domains/create' page={CreateDomainPage} />
+        <Route path='/domains/create' page={DomainCreatePage} />
         <Route path='/domains/:domainId' page={DomainPage} />
         <Route path='/domains/:domainId/links/:linkId' page={LinkPage} />
         <Route path='/domains/:domainId/links/create' page={CreateLinkPage} />
```

Comment:
actually on second thoughts, we should define a convention and stick to it. the alternative for this is DomainLinkCreate page

Reply:
yep will update in that PR (hopefully later today) 👍